### DISCUSSION TITLE
Oracle Cloud API Authentication

### DISCUSSION THREAD

**Discussion by brasrox** [grafana/grafana-infinity-datasource#873] [Ideas]
Is it possible to have the ability to authenticate in Oracle Cloud?
it would be great to be able to use their rest api to get the daily cost in grafana
https://docs.oracle.com/en-us/iaas/api/#/en/usage/20200107/

its use a signingrequests like 
https://docs.oracle.com/en-us/iaas/Content/API/Concepts/signingrequests.htm

**Comment by yesoreyeram** [comment]
No. It is not currently supported.
